src/Model/ActivityList/DefaultMariaDbActivityList.php

```diff
@@ -116,7 +117,7 @@ public function getPaginatorAdapter()
         throw new \Exception('Not implemented anymore.');
     }
 
-    public function current(): ActivityInterface | false
+    public function current(): ActivityInterface | DefaultActivityStoreEntry | false
```

Comment:
```suggestion
    public function current(): ActivityStoreEntryInterface | false
```
Does this work? as [createEntryInstance](https://github.com/pimcore/customer-data-framework/blob/f24b239e62717fdd4a1eb1d8f55c26b39414e0e4/src/Model/ActivityList/DefaultMariaDbActivityList.php#L161) creates an instance of ActivityStoreEntryInterface